libs/framework/gtest/CMakeLists.txt

```diff
@@ -15,6 +15,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
```

Comment:
It seems that a `CELIX_CXX17` testing is missing.

Reply:
correct, added the check in the framework CMakeLists.txt